commercetools-models/src/test/java/io/sphere/sdk/carts/queries/CartQueryIntegrationTest.java

```diff
@@ -265,4 +267,31 @@ public void locale() {
         assertThat(result.getResults()).hasSize(1);
         assertThat(result.head().get().getId()).isEqualTo(cart.getId());
     }
+
+    @Test
+    public void inStoreLocale() {
+        StoreFixtures.withStore(client(), store -> {
```

Comment:
Here it would be nice to use a static import - that would make it more consistent with the other code 😉